docs/source/get_involved/governance.rst

```diff
@@ -324,6 +357,8 @@ expected to resign voluntarily.
 Tenure also ends automatically when a CC member's tenure as core developer ends,
 e.g., through resignation or inactivity.
 
+CC members can be removed from the CC by an 80% majority of CC members.
+
```

Comment:
```suggestion
CC members can be removed from the CC by an 80% majority of core-developers.

```

Reply:
@aiwalter maybe "by an 80% majority of all cast votes"?

Reply:
I think thats same bcz CC members are also core-devs always?

Reply:
Not sure about the 80%, probably a 2/3 majority should be better?

Reply:
since this is unrelated to CoCC functionality and may require some additional discussion, I moved it out into a new PR https://github.com/sktime/sktime/pull/3697, and added some details.